changelog/unreleased/issue-20223.toml

```
@@ -0,0 +1,5 @@
+type="a"
+message="Added new Palo Alto version 11 input to ingest PAN OS 11 logs."
```

Nit, but I would suggest using the exact input name in the change log for clarity.
```suggestion
message="Added new Palo Alto Networks TCP (PAN-OS v11+) input."
```